packages/ui/react/package.json

```diff
@@ -57,8 +57,8 @@
   },
   "homepage": "https://github.com/devopness/devopness#readme",
   "dependencies": {
-    "@mui/material": "^6.4.11",
-    "@mui/styled-engine-sc": "^6.4.9",
+    "@mui/material": "^7.1.0",
+    "@mui/styled-engine-sc": "^7.1.0",
     "ldrs": "^1.1.7",
```

Comment:
No components needed to be updated?
Could add evidence on the PR description of running Storybook locally with all components rendered properly after upgrade material UI to v7?

Reply:
@lucasbachega got it.

> Could you please add evidence on the PR description of running Storybook locally with all components rendered properly after upgrade material UI to v7?

Regarding this ... could you update the PR description and resolve the conflicts in this PR?